lib/azure/resource_management/ARM_interface.rb

```diff
@@ -156,19 +159,41 @@ def show_server(name, resource_group)
 
             details << ui.color('OS Type', :bold, :cyan)
             details << server.properties.storage_profile.os_disk.os_type
-      
+
             puts ui.list(details, :columns_across, 2)
-            
+
          else
            puts "There is no server with name #{name} or resource_group #{resource_group}. Please provide correct details."
-         end  
-       
+         end
+
         rescue => error
           puts "#{error.class} and #{error.message}"
         end
 
-      end 
-     
+      end
+
+      def delete_resource_group(resource_group_name, vm_name)

```

Comment:
It would be good to keep `delete_resource_group` method independent of `vm_name` so that this methods takes care of only resource_group deletion. We should keep VM deletion process separate.